src/main/java/wolox/training/models/Users.java

```diff
@@ -0,0 +1,75 @@
+package wolox.training.models;
+
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.JoinTable;
+import javax.persistence.ManyToMany;
+import wolox.training.exceptions.BookAlreadyOwnedException;
+
+@Entity
+public class Users {
+	@Id
+	@GeneratedValue(strategy = GenerationType.AUTO)
+	private long id;
+
+	@Column(nullable = false)
+	private String userName;
+
+	@Column(nullable = false)
+	private String name;
+
+	@Column(nullable = false)
+	private LocalDate localDate;
+
+	@ManyToMany(cascade = CascadeType.ALL)
+	@JoinTable(name = "book_users",
+			joinColumns = @JoinColumn(name = "book_id", referencedColumnName = "id"),
+			inverseJoinColumns = @JoinColumn(name = "users_id",
+					referencedColumnName = "id"))
+	private List<Book> books;
+
+	public Users(final String userName, final String name, final LocalDate localDate) {
```

Comment:
When you create a constructor with arguments, you need to add an **empty constructor**, which is required by JPA to allow creation of instances via reflection (otherwise, when instantiating an object for persistence, JPA can't figure out how to invoke the constructor because it can't make up values for the parameters)